templates/CONTRIBUTING.md

```diff
@@ -203,32 +213,34 @@ requirements:
     - 1 or more [objective files](#obj-files)
     - 1 or more [user task files](#user-task-files)
     - 1 or more [requirement files](#req-files)
-- Required sections:
-    - yaml index header, including keys for `title:`, `persona:`, `objective:`, `user_tasks:`, and `requirements:`
+- **Required sections:**
+    - yaml index header, including keys for `title:`, `completed:`, `tutorial:`, `goals:`, `persona:`, `objective:`, `user_tasks:`, and `requirements:`. The `completed:`, `tutorial:`, and `goals:` keys should be left empty; the Use Case committee will add information during the [approval process](#approval-process).
     - a short summary of the use case
 
 #### `Persona` files <a name="persona-files"></a>
 - Use the [persona file template](https://github.com/nih-cfde/use-case-library-build/tree/latest/library_file_templates/library-persona-template.md) to enter the required sections
-- Required sections:
+- **Required sections:**
     - yaml index header with `title:`
     - Short description of the persona's biological/computational experience, their role and responsibilities, and relation to any of the other personas in the Use Case Library
     - A bullet point section listing assumptions about the persona's credentials e.g., access to the CFDE
```

Comment:
Assumption are just assumptions in general, not specific to credentials

Reply:
updated